src/RequestHandler/outofprocess/forwardinghandler.h

```diff
@@ -174,11 +174,33 @@ class FORWARDING_HANDLER : public REQUEST_HANDLER
     BOOL                                m_fResponseHeadersReceivedAndSet;
     BOOL                                m_fResetConnection;
     BOOL                                m_fHandleClosedDueToClient;
-    BOOL                                m_fFinishRequest;
-    BOOL                                m_fHasError;
     BOOL                                m_fDoReverseRewriteHeaders;
+    BOOL                                m_fServerResetConn;
+    volatile  BOOL                      m_fClientDisconnected;
```

Comment:
Cannot find where m_fClientDisconnected is set?

Reply:
fixed